docs/api/list-kibana652.json

```diff
@@ -4,6 +4,7 @@
     "download": "/package/example-0.0.5.tar.gz",
     "icon": "/img/example-0.0.5/icon.png",
     "name": "example",
+    "title": "",
```

Comment:
@jfsiii This is the way it will look like at the moment if the title is not set. The field is in the response but with an empty string. Is that what you expected or better the entry does not exist at all?

Reply:
My preferences are

1. No `title` entry
1. `title` value of `undefined`
1. `title` value of `""` (empty string)

but any of these is workable

Reply:
Option 1 it is: https://github.com/elastic/integrations-registry/pull/50

Reply:
The `/list` and `/package` endpoint have slightly different behaviors.

The first result from `curl http://integrations-registry.app.elstc.co/list` has no `title` key:

```json
  {
    "description": "Mysql Logs and metrics",
    "download": "/package/mysql-1.0.2.tar.gz",
    "icon": "/img/mysql-1.0.2/icon.png",
    "name": "mysql",
    "version": "1.0.2"
  },
```


but `curl http://integrations-registry.app.elstc.co/package/mysql-1.0.2` has `title` key and `null` value:

```json
{
  "name": "mysql",
  "title": null,
  "version": "1.0.2",
  "description": "Mysql Logs and metrics",
  "icon": "/img/mysql-1.0.2/icon.png",
  "requirement": {
    "kibana": {
      "version.min": "6.7.0",
      "version.max": "7.6.0"
    }
  }
}
```

Reply:
Related: I think many of the titles should be updated (e.g. "MySQL" , "PostgreSQL", "HAProxy", "iptables", etc). Should I open an issue, comment here, or something else?

Reply:
Comment here works, will fix it.

Reply:
Here is the fix: https://github.com/elastic/integrations-registry/pull/60 I left out some titles on purpose so you can test on how it works without ;-)